arithmetization/src/main/java/net/consensys/linea/sequencer/txselection/selectors/TraceLineLimitTransactionSelector.java

```diff
@@ -157,4 +161,25 @@ private String logTxLineCount() {
                     + moduleLimits.get(e.getKey()))
         .collect(Collectors.joining(",", "[", "]"));
   }
+
+  private class ZkTracerWithLog extends ZkTracer {
+    @Override
+    public void traceEndBlock(final BlockHeader blockHeader, final BlockBody blockBody) {
+      super.traceEndBlock(blockHeader, blockBody);
+      log.atTrace()
+          .addMarker(BLOCK_LINE_COUNT_MARKER)
+          .addKeyValue("blockNumber", blockHeader::getNumber)
+          .addKeyValue("blockHash", blockHeader::getBlockHash)
+          .addKeyValue(
+              "traceCounts",
+              () ->
+                  currCumulatedLineCount == null
+                      ? "null"
+                      : currCumulatedLineCount.entrySet().stream()
+                          .sorted(Map.Entry.comparingByKey())
+                          .map(e -> '"' + e.getKey() + "\"=" + e.getValue())
+                          .collect(Collectors.joining(",")))
+          .log();
```

Comment:
Is the output here different from logTxLineCount()? could we merge them?

Reply:
they cannot be merge, since are meant for different purposes, the `logTxLineCount` is at tx level, and meant for debugging and to be consumed by humans, while this one is a request from @FlorianHuc to match the format of zkGeth to be able to compare and monitor the result with a tool